### ISSUE TITLE
Editor shows error for multiline matcher

### ISSUE THREAD

**Issue by kdebski85** [will7200/casbin-idea-plugin#12]
For multiline matcher, the editor provided by the plugin shows error on `&& \` : `<expr> expected, got '\'`.
The matcher works perfectly fine. The only issue is the error in the editor.

Sample `model.conf`:

```
[request_definition]
r = foo, bar, xyz

[policy_definition]
p = foo, bar, xyz, eft

[policy_effect]
e = some(where (p.eft == allow)) && !some(where (p.eft == deny))

[matchers]
m = r.foo == p.foo && \
    r.bar == p.bar && \
    r.xyz == p.xyz
```

**Comment by hsluoyz** [comment]
@will7200

**Comment by will7200** [comment]
Looking into this

**Comment by will7200** [comment]
Made a fix in 2de374df66f0fb4469bf9cfaa1673511b1db0885 will release to jetbrains for review shortly
